libraries/c_sdk/aws/defender/test/system/aws_iot_tests_defender_system.c

```diff
@@ -516,10 +533,16 @@ static IotMqttError_t _startMqttConnection( void )
         mqttConnectionInfo.pClientIdentifier = AWS_IOT_TEST_DEFENDER_THING_NAME;
         mqttConnectionInfo.clientIdentifierLength = ( uint16_t ) strlen( AWS_IOT_TEST_DEFENDER_THING_NAME );
 
-        mqttError = IotMqtt_Connect( &mqttNetworkInfo,
-                                     &mqttConnectionInfo,
-                                     1000,
-                                     &_mqttConnection );
+        /* AWS IoT Service limits only allow 1 connection per MQTT client ID per second.
+         * Wait until 1100 ms have elapsed since the last connection. */
+        uint32_t periodMs = 1100;
```

Comment:
Since this file is from the C SDK, variables [should be](https://docs.aws.amazon.com/freertos/latest/lib-ref/c-sdk/main/guide_developer_styleguide.html) declared at the top of a function.